doc/quickstart.md

```diff
@@ -0,0 +1,98 @@
+
+This guide provides a brief summary of instructions to set up reliable
+benchmark measurements using BenchExec and important points to consider. It is
+meant for users who either want to set up benchmarking from scratch or already
+have a simple setup using, e.g., `time`, `timeout`, `taskset`, `ulimit`, etc.
+and will guide you how to use `runexec` as a simple but much more reliable
+"drop-in" replacement for these tools.
+
+## Guiding Example
+
+> [!IMPORTANT]
+> If your current setup looks similar to the below example (or you are thinking
+> about such a setup), we strongly recommend following this guide for a much
+> more reliable process.
+
+As an example, suppose that you want to measure the performance of your
+tool `program` with arguments `--foo` and `--bar` on the input files
+`input_1.in` to `input_9.in`. To measure the runtime of the tool, one may run
+```
+$ /usr/bin/time program --foo input_1.in
+$ /usr/bin/time program --bar input_1.in
+$ /usr/bin/time program --foo input_2.in
+...
+```
+etc. and note the results. In case resource limitations are desired (e.g. 
+limiting to 1 CPU and 60 sec of wallclock time), the calls might be
+```
+$ taskset -c 0 timeout 60s /usr/bin/time program ...
+```
+or similar.
+
+## Benchmarking with BenchExec
+
+The following steps guide you to increase the reliability and quality of
+measurements drastically by using BenchExec instead of these small
+utilities.
+
+### Step 1. Install BenchExec
+
+Depending on your distribution, the setup steps vary slightly.
+On Debian/Ubuntu and similar, you can
+```
+sudo add-apt-repository ppa:sosy-lab/benchmarking
+sudo apt update && sudo apt install benchexec
+```
+Otherwise, try the pip-based setup
+```
+pip3 install --user benchexec[systemd] coloredlogs
+```
+
+You should be able to run the command
+`python3 -m benchexec.check_cgroups` without issues. See
+[the installation guide](INSTALL.md) for further details and troubleshooting.
+
+### Step 2. Consider the Setup of Your Benchmark
+
+Consider and document the benchmarking process *beforehand*. In particular,
+think about which executions you want to measure, what resource limits should
+be placed on the benchmarked tool(s), such as CPU time, CPU count, memory, etc.
+Also consider how timeouts should be treated.
+
+For more complicated setups, please also refer to the
+[benchmarking setup guide](benchmarking.md). For example, in case you want to
+execute multiple benchmarks in parallel, think about how to deal with shared
+resources (e.g. the memory bus and CPU cache). For such cases, we recommend
+using `benchexec` instead, which takes care of managing parallel invocations.
+
+
+### Step 3. Gather Measurements using runexec
+
+Using the example from above, suppose that we want to limit the process to 60s
+wall time, 1 GB of memory, and cpu core 0. Then, simply run
+```
+$ runexec --quiet --walltimelimit 60s --memlimit 1GB --cores 0 --output output_1_foo.log -- \
```

Comment:
I would remove `--quiet` and `--output` here. The less options, the better.

Reply:
Agree with quiet but without --output the run output is put into `output.log` which can be unexpected (overwriting the previous outputs)

Reply:
On this note: Would it be useful to maybe have `output-%Y%M%D-%h%m%s.log` (or similar)  as default?

Reply:
Good question. Then we would have to do the same for `output.files`, of course. I am not sure how useful it is, because if you want to do real benchmarking, just knowing the time stamp will not be enough, you need more metadata anyway, but it likely wouldn't hurt. But we could change this only for the next major version of BenchExec.

Reply:
Arguably, if you do "real benchmarking" you probably set the output path manually too ;)

On that note, there might also be a point in actually printing that meta information too. Concretely, this would probably mean generating a unique identifier for each execution (e.g. id = time_random number, or even just time up to milliseconds) and then writing out `run_<id>.out` `run_<id>.err` and `run_<id>.meta`, where the "meta" file contains everything relevant (e.g. things like version of benchexec, the complete command line invocation, cwd, env, uname, and maybe the run results?)

(this is not the right place to discuss this, if this sounds interesting I can create an issue for this)